roles/ci_gen_kustomize_values/tasks/generate_values.yml

```diff
@@ -64,16 +73,30 @@
   loop_control:
     label: "{{ item.source | basename }}"
 
-- name: Output values file
-  ansible.builtin.copy:
-    dest: >-
+- name: Generate values.yaml equivalent
+  vars:
+    _destdir: >-
       {{
         (cifmw_ci_gen_kustomize_values_generated_dir,
-        (values_datatype | default('')),
-        cifmw_ci_gen_kustomize_values_dest_filename) | path_join
+         values_datatype | default('')
```

Comment:
Aren't we expecting `values_datatype` to be defined always? Cause if that's the case a default here will hide an actual clean error.

Reply:
it's mostly when if we're calling that task files without calling the gen_snippet first. It would be weird though....